ydb/core/kqp/ut/data_integrity/kqp_data_integrity_trails_ut.cpp

```diff
@@ -137,6 +137,67 @@ Y_UNIT_TEST_SUITE(KqpDataIntegrityTrails) {
         // check datashard logs (should be empty, because DataShard only logs modification operations)
         UNIT_ASSERT_VALUES_EQUAL(CountSubstr(ss.Str(), "DATA_INTEGRITY INFO: Component: DataShard"), 0);
     }
+
+    Y_UNIT_TEST(UpsertViaYqlScripting) {
+        TKikimrSettings serverSettings;
+        TStringStream ss;
+        serverSettings.LogStream = &ss;
+        TKikimrRunner kikimr(serverSettings);
+        kikimr.GetTestServer().GetRuntime()->SetLogPriority(NKikimrServices::DATA_INTEGRITY, NLog::PRI_TRACE);
+        NYdb::NScripting::TScriptingClient client(kikimr.GetDriver());
+
+        auto result = client.ExecuteYqlScript(R"(
+            --!syntax_v1
+
+            UPSERT INTO `/Root/KeyValue` (Key, Value) VALUES
+                (3u, "Value3"),
+                (101u, "Value101"),
+                (201u, "Value201");
+        )").GetValueSync();        
+        UNIT_ASSERT_VALUES_EQUAL_C(result.GetStatus(), EStatus::SUCCESS, result.GetIssues().ToString());
+            
+        // check executer logs (should be empty, because executer only logs modification operations)
```

Comment:
```suggestion
        // check executer logs
```

cause UNIT_ASSERT(..., 1);